src/screens/OldHomepage/index.js

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
```

Comment:
Can remove this?

Reply:
Yeah I can remove the entire OldHomepage dir.